backend/api/Services/EchoService.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System.Net.Http.Headers;
+using System.Text.Json;
+using System.Web;
+using Api.Models;
+using Azure.Core;
+using Azure.Identity;
+
+namespace Api.Services
+{
+    public class EchoService
+    {
+        private readonly ILogger<EchoService> _logger;
+
```

Comment:
No need for empty line between private class properties imo, open for discussion

Reply:
Agree